.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
This is strange phrasing IMO. Especially considering that we're in the `wdl` crates which are a library

Reply:
That's a good point. I just copied this over from: https://github.com/stjudecloud/bioinformatics-tool-template/blob/main/.github/ISSUE_TEMPLATE/bug_report.md. Now I've made some edits to customize it for the `wdl` crates.